subgraph-matic.yaml

```diff
@@ -52,6 +52,9 @@ dataSources:
       abis:
         - name: RealmDiamond
           file: ./abis/RealmDiamond.json
+      callHandlers:
+        - function: setParcelsAccessRightWithWhitelists(uint256[],uint256[],uint256[],uint32[])
+          handler: handleSetParcelsAccessRightWithWhitelists
```

Comment:
We should also add the handler that set the parcel access rights without the whitelist